service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandler.java

```diff
@@ -49,4 +57,19 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
         notificationService.sendEmail(STANDARD_DIRECTION_ORDER_ISSUED_TEMPLATE, email, parameters,
             eventData.getReference());
     }
+
+    @EventListener
+    public void notifyAllocatedJudgeOfIssuedStandardDirectionsOrder(StandardDirectionsOrderIssuedEvent event) {
+        if (featureToggleService.isSDONotificationForAllocatedJudgeEnabled()) {
+            EventData eventData = new EventData(event);
+            CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+            Map<String, Object> parameters = standardDirectionOrderIssuedEmailContentProvider
+                .buildNotificationParametersForAllocatedJudge(eventData.getCaseDetails());
+
+            String email = caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor().getJudgeEmailAddress();
+
+            notificationService.sendEmail(STANDARD_DIRECTION_ORDER_ISSUED_JUDGE_TEMPLATE, email, parameters,
+                eventData.getReference());
+        }
+    }
```

Comment:
Hmmm, could we have the feature toggle check in the controller itself? That way we don't even create the event object if condition is not met.

Reply:
Discussed why we shouldn't do this for sdo - same event is triggered already for toggled on notifications.. but i will do this for Notice of proceedings as single event - > notification